src/Generators/Webserver/Database/Drivers/PostgresSchema.php

```diff
@@ -54,6 +54,6 @@ public function updated(Updated $event, array $config, Connection $connection):
 
     protected function dropDatabase(IlluminateConnection $connection, array $config)
     {
-        return $connection->statement("DROP SCHEMA IF EXISTS \"{$config['schema']}\"");
+        return $connection->statement("DROP SCHEMA IF EXISTS \"{$config['schema']}\" CASCADE");
```

Comment:
Using `CASCADE` is quite a big danger it, as far as my PSQL knowledge goes, this would delete everything in the schema. I think it would be better to use `RESTRICT` **if** we should change this at all. I think `RESTRICT` is even the default.
If we change this, we could risk people ending up deleting entire schema's while that's something that they would not want.

Reply:
My thoughts on this method is that we shouldn't be executing this code if our intent is not to drop the schema. And if our intent is to do so, why wouldn't we follow through with it? The problem I ran into when not adding the CASCADE is that the schema would never be dropped automatically, because the tables it contained were never dropped.